Jenkinsfile

```diff
@@ -1,77 +1,89 @@
-//configuration
 properties([[$class: 'jenkins.model.BuildDiscarderProperty', strategy:
-			[$class: 'LogRotator', numToKeepStr: '100', artifactNumToKeepStr: '20']
-			]])
-			
-
-node {
-    echo "current branch: ${env.BRANCH_NAME}"
-}
-
+            [$class: 'LogRotator', numToKeepStr: '100', artifactNumToKeepStr: '20']
+            ]])
+            
 pipeline {
     agent any
+
     stages {
-        stage('Setup') {
+        stage('Build') {
             steps {
-                
-                sh "git submodule init" 
-
-                sh "git submodule update --init --recursive"
-
-                sh "./gradlew build pack"
+                echo "Building branch: ${env.BRANCH_NAME}"
+                sh "./gradlew pack" 
             }
-            
         }
         
         stage('Archive build output') {
             when {
-                expression { GIT_BRANCH == 'master' || GIT_BRANCH == 'dev' || GIT_BRANCH == 'ci' || GIT_BRANCH == 'dev-audit' }
+                expression { 
+                    GIT_BRANCH == 'master' ||
+                        GIT_BRANCH == 'dev' ||
```

Comment:
could we have `master-pre-merge` here as well? it could actually replace `dev` which is not used

Reply:
Sure.  BTW just so there's no confusion, this is archiving the .tar.bz2 produced by pack; I don't think this archives things like test result or Jenkins workspace.

Reply:
I will add master-pre-merge and remove ci, dev, dev-audit (from discussions in Slack, those two are also not needed)